tests/vnc_test.go

```diff
@@ -114,6 +114,70 @@ var _ = Describe("[rfe_id:127][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				Expect(response).To(Equal("RFB 003.008\n"))
 				Expect(err).To(BeNil())
 			})
+
+			It("should allow accessing the VNC device multiple times", func() {
```

Comment:
So much duplicate code...
Can we put the complete test function from the first It() above into a variable, use it directly in the It() above, and here in a loop?

Reply:
Hey Marc. I have united the code. Now it is reusing the whole subroutine.

Reply:
perfect :+1: